setup/generate.ps1

```diff
@@ -0,0 +1,142 @@
+<#
+.SYNOPSIS
+    Generate the Azure Cmdlets MSI files.
+.DESCRIPTION
+    This script installs the azure cmdlets from the powershell gallery, tweaks them and generates MSI files for x86 and x64.
+    When the MSI installs it will install the cmdlets to the same location the gallery would, and PowerShellGet treats them 
+    as if they were installed from the gallery
+.PARAMETER Version
+    The version number for the generated MSI.
+.PARAMETER Force
+    Forces a fresh installation of the Azure and AzureRm cmdlets from the gallery
+.PARAMETER noBuildNumber
+    Prevent a build number from being tacked on the end of the version number.
+.PARAMETER repository
+    Set the repository to pull packages from.
+#>
+
+param(
+    [Parameter(HelpMessage="The version number for the generated MSI.")]
+    [string]$version = "6.0.0",
```

Comment:
I really don't think we should have a default version, we should have to manually enter it since 6.0.0 will only be right once.